stanley/helpers.py

```diff
@@ -58,7 +58,8 @@ def get_receiver(members, sender):
 
     # if the size of the list is zero, everyone received a feedback and we can start again
     if len(can_receive) == 0:
-        redis_storage.delete(REDIS_KEY_SEND_FEEDBACK)
```

Comment:
I found this a bug (the use of `REDIS_KEY_SEND_FEEDBACK` instead of `REDIS_KEY_RECEIVE_FEEDBACK`) but I'd like to confirm with you first. Let me know. :)